group-coordinator/src/test/java/org/apache/kafka/coordinator/group/classic/ClassicGroupTest.java

```diff
@@ -1459,6 +1459,7 @@ public void testFromConsumerGroupWithJoiningMember() {
 
         ClassicGroup classicGroup = ClassicGroup.fromConsumerGroup(
             consumerGroup,
+            Collections.emptySet(),
```

Comment:
Not related to this line, should we rename the method `testFromConsumerGroupWithReplacingMember`?

Reply:
Sounds reasonable, did the rename